ansible/roles/runner/tasks/deploy_pr_ci.yml

```diff
@@ -6,16 +6,6 @@
     version: "{{ pr_ci_repo_branch }}"
     force: true
 
-- name: install python dependencies with pip
-  pip:
-    executable: pip3
-    state: latest
-    name: "{{ item.name }}"
-  with_items:
-    - name: "setuptools"
-    - name: "github3.py>=1.0.0a4"
-    - name: "CacheControl>=0.12.3"
```

Comment:
There is a good reason the minimal version of CacheControl is `>=0.12.3`. I don't remember the details, but there were some intricate caching issues.

Until Fedora 28, you have two options: get it from pip, or use my copr repo [tkrizek/freeipa-pr-ci](https://copr.fedorainfracloud.org/coprs/tkrizek/freeipa-pr-ci/) (this is how it used to be).

Tracked bug for rebase: https://bugzilla.redhat.com/show_bug.cgi?id=1484313